easybuild/easyconfigs/l/Libint/Libint-2.0.3-foss-2018b.eb

```diff
@@ -0,0 +1,19 @@
+name = 'Libint'
+version = '2.0.3'
+
+homepage = 'https://sourceforge.net/p/libint'
+description = """Libint library is used to evaluate the traditional (electron repulsion) and certain novel two-body
+ matrix elements (integrals) over Cartesian Gaussian functions used in modern atomic and molecular theory."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+toolchainopts = {'pic': True}
+
+source_urls = ['http://downloads.sourceforge.net/project/libint/libint-for-mpqc']
+sources = ['libint-%(version)s-stable.tgz']
+patches = ['Libint-%(version)s-stable-fma.patch']
+checksums = [
+    '5d4944ed6e60b08d05b4e396b10dc7deee7968895984f4892fd17159780f5b02',  # libint-2.0.3-stable.tgz
+    '948388d66e898fb8fdb4964b7852fd7d13999bcfb97339bf8e69ef471ca982b1',  # Libint-2.0.3-stable-fma.patch
```

Comment:
@tovrstra Patch checksum was not updated with last change to patch (adding author)

Reply:
omg, sorry for not paying attention. should be fixed.